sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -3183,6 +3192,8 @@ class SQLConf extends Serializable with Logging {
 
   def nestedSchemaPruningEnabled: Boolean = getConf(NESTED_SCHEMA_PRUNING_ENABLED)
 
+  def optimizerIgnoreHints: Boolean = getConf(OPTIMIZER_IGNORE_HINTS)
```

Comment:
nit: I think the new policy is to not add methods to `SQLConf` if it's only called once.